specification/paths/Organizations-organization_id.json

```diff
@@ -23,7 +23,32 @@
               "additionalProperties": false,
               "properties": {
                 "data": {
-                  "$ref": "#/components/schemas/Organization"
+                  "allOf": [
+                    {
+                      "$ref": "#/components/schemas/Organization"
+                    },
+                    {
+                      "required": [
+                        "attributes",
+                        "relationships"
+                      ],
+                      "properties": {
+                        "id": {
+                          "readOnly": true
```

Comment:
The `id` in the `get` and `patch` endpoints should be required. I don't know if they are already required, but in swagger ui it looks like they are not.